sycl/source/detail/kernel_bundle_impl.hpp

```diff
@@ -192,6 +192,7 @@ class kernel_bundle_impl {
     // TODO: Unify with c'tor for sycl::comile and sycl::build by calling
     // sycl::join on vector of kernel_bundles
 
+    // The loop below just links each device image separately, not linking any two device images together. This is correct so long as each device image has no unresolved symbols. That's the case when device images are created from generic SYCL APIs. There's no way in generic SYCL to create a kernel which references an undefined symbol. If we decide in the future to allow a backend interop API to create a "sycl::kernel_bundle" that references undefined symbols, then the logic in this loop will need to be changed.
```

Comment:
I presume we have some line length limit in our source base, so this should be split into multiple lines?

Reply:
No, we rely on clang-format and our dev team to remember to ADD the clang-format modification back into their commits.  I ran clang-format, but forgot to adjust the commit with it. Just bumped it.  Thanks.